tests/unit/amo/reducers/test_home.js

```diff
@@ -90,13 +98,29 @@ describe(__filename, () => {
       ]);
     });
 
+    it('loads hero shelves', () => {
+      const { store } = dispatchClientMetadata();
+
+      const heroShelves = createHeroShelves();
+      _loadHomeData({
+        store,
+        heroShelves,
+      });
+
+      const homeState = store.getState().home;
+
+      expect(homeState.heroShelves).toEqual(
+        createInternalHeroShelves(heroShelves),
+      );
```

Comment:
I'd like to see more test cases for `createInternalHeroShelves` when:

- `addon` is not present
- `external` is not present
- `description` is `null` in `primary`
- the main `cta` is `null` in `secondary`
- one module has a `cta` that is `null` in `secondary`